bedrock/firefox/templates/firefox/whatsnew/fx54/whatsnew-54.html

```diff
@@ -0,0 +1,126 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{#
+A modified copy of firefox/mobile-download-desktop.html
+#}
+
+{% from "macros.html" import google_play_button, send_to_device with context %}
+
+{% add_lang_files "firefox/whatsnew_50" "firefox/sendto" %}
+
+{% extends "firefox/base-resp.html" %}
+
+{% block gtm_page_id %}data-gtm-page-id="/firefox/whatsnew/"{% endblock %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{% block page_title %}{{_('Download Firefox for Android and iOS')}}{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
+{% endblock %}
+
+{#- This will appear as <meta property="og:image"> which can be used for social share -#}
+{% block page_image %}
+  {{ static('img/firefox/firefox-independent-1200.jpg') }}
+{% endblock %}
+
+{% block body_id %}firefox-mobile-download{% endblock %}
+{% block body_class %}{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block page_css %}
+  {% stylesheet 'firefox_whatsnew_54' %}
+{% endblock %}
+
+{% set show_send_to_device = LANG in settings.SEND_TO_DEVICE_LOCALES %}
+
+{% set send_to_device_message_set = 'fx-50-whatsnew' %}
+{% set app_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&amp;adgroup=iOS_App_Store&amp;creative=badge_en&amp;fallback=https%3A%2F%2Fitunes.apple.com%2Fapp%2Fapple-store%2Fid989804926%3Fpt%3D373246%26ct%3Dadjust_tracker%26mt%3D8' %}
+{% set play_store_url = 'https://app.adjust.com/2uo1qc?campaign=whats_new&amp;adgroup=Google_Play_Store&amp;creative=badge_en' %}
+
+{% block content %}
+<main role="main">
+  <div class="main-container">
+    <div class="horizon">
+      <div class="stars">
+        <div class="inner-container">
+          <header id="masthead">
+            <div id="tabzilla">
```

Comment:
I know this is just reusing an old template, but can we just remove the old logo here? Seems a shame to be showing it on /whatnew